### ISSUE TITLE
amp-story: Video/animations cannot be paused or stopped

### ISSUE THREAD

**Issue by TetraLogicalHelpdesk** [ampproject/amphtml#32509]
# Video/animations cannot be paused or stopped

## WCAG Level
Level A

## Priority
High

## Pages/screens/components affected
* [Navigation](https://www.gstatic.com/amphtml/stamp/qa/navigation.html)
* [Video](https://www.gstatic.com/amphtml/stamp/qa/video.html)
* [Animation](https://www.gstatic.com/amphtml/stamp/qa/animation.html)
* [Prerender - Video](https://www.gstatic.com/amphtml/stamp/qa/prerender-video.html?amp_js_v=0.1#origin=https://www.google.com&prerenderSize=1&visibilityState=prerender&paddingTop=0&p2r=0&horizontalScrolling=0&csi=1&aoh=15526927198732&viewerUrl=https://www.google.com&history=1&storage=1&cid=1&cap=swipe,navigateTo,cid,fragment,replaceUrl)
* [AMP Story Player - AMP Story 360](https://www.gstatic.com/amphtml/stamp/qa/amp-story-360.html)

## Description
When a page contains autoplaying video or animations that last for longer than 5 seconds, users must be able to pause/stop/hide the content.

On large screen/desktop, when an AMP story page contains a video, the general "Pause" control in the top-right corner can be used to pause the video content of the current page (as well as suppressing any auto-advancing on the page itself).

![An AMP Story page with video running, showing the pause control in the top-right corner](https://user-images.githubusercontent.com/54933393/107249895-a6d44800-6a2b-11eb-8c6a-a9b32c9bad82.png)

However, in the small-screen/smartphone sized version, there is no "Pause" control visible. Desktop users that have sized/zoomed their browser window to a small-enough size to trigger the "smartphone" view have no way of pausing the video with their mouse or keyboard. The same problem applies to users on an actual smartphone that are using an external mouse or keyboard.

![The same AMP Story page with video, in a small-enough desktop browser window, lacking a pause control](https://user-images.githubusercontent.com/54933393/107250128-b8b5eb00-6a2b-11eb-8418-b3d64da1561b.png)

For pages that use animation (rather than video) - such as [Animation](https://www.gstatic.com/amphtml/stamp/qa/animation.html) - there is no pause control visible on large-screen/desktop, making it impossible for keyboard/mouse users to pause the animation in both the large and small screen version.

![An AMP Story page with an animation, on desktop, showing no pause control](https://user-images.githubusercontent.com/54933393/107250335-c79c9d80-6a2b-11eb-9b46-bde78c29caf0.png)

Touchscreen users are able to pause video and animations in the small-screen/smartphone view by pressing and holding on the video/animation itself. While nominally this passes the requirement of having a method to pause/stop/hide the content, this is - subjectively - not a good approach, as it forces users to keep their finger pressed on the touchscreen to prevent video/animations from playing.

## User impact
Someone with a vestibular disorder may find that the constantly updating carousel makes them feel nauseous, whereas someone who finds reading difficult, perhaps because they have Dyslexia, will struggle to read the content of each block before it is replaced by the next.

## Required solution
Make sure that on both large and small screen view, there is a method for users keyboard, mouse, and touch users to pause both videos and animations.

In particular, in the small-screen view, make the pause control visible and actionable the same way that it appears on large-screen view. Additionally, make sure that the pause control also appears for animations, not just when a page contains a video element (or is set to auto-advance).

This solution must be applied to all instances of the issue identified within the test sample, then applied to all other instances of the same issue identified throughout the rest of the website.

## Test procedure(s)
Use these steps to confirm that the solution has been correctly applied to issues identified within the test sample, and to test the rest of the website for instances of the same issue:
1. Check that the carousel does not automatically update.
2. Check that there are controls to let users move backwards and forwards through the carousel.
3. If the carousel must update automatically, check there is a button to let users stop and restart it.

### Definition of done
Complete all of these tasks before closing this issue or indicating it is ready for retest:
* All issues identified within the test sample have been resolved.
* The rest of the website has been tested for the same issue.
* All issues identified throughout the rest of the website have been resolved or filed as new issues.

## Related standards
* [WCAG 2.1 Success Criterion 2.2.2 Pause, Stop, Hide (Level A)](https://www.w3.org/TR/WCAG21/#pause-stop-hide)

## More information
* [W3C - Understanding Success Criterion 2.2.2: Pause, Stop, Hide](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html)
* [W3C - Technique G186: Using a control in the Web page that stops moving, blinking, or auto-updating content](https://www.w3.org/WAI/WCAG21/Techniques/general/G186)
* [W3C - Technique G4: Allowing the content to be paused and restarted from where it was paused](https://www.w3.org/WAI/WCAG21/Techniques/general/G4)

## Test data
Test date: January 2021

**Comment by gmajoulet** [comment]
Duplicate of https://github.com/ampproject/amphtml/issues/32508 as they're both related to how to pause the story's content.
